packages/svelte/src/index.js

```diff
@@ -0,0 +1,57 @@
+async function htmlLoader(req, res, next) {
+  const { ResolvedComponent, componentProps, component } = req.locals;
+
+  try {
+    const { html, css } = ResolvedComponent.render(componentProps);
+
+    res.locals.html = `
+          <style>${css.code}</style>
+          <div
+              data-ssr-aas-component="${component}"
+              data-ssr-aas-id="se_embed_svelte_ssr_${req.id}"
+          >${html}</div>
+          <script>
+              window.__SSR_AAS_LOAD_PROPS__ ||= {};
+              window.__SSR_AAS_LOAD_PROPS__[${req.id}] = ${JSON.stringify(
+  componentProps,
+  null,
+  3,
+)};
+          </script>
+      `;
+
+    next();
+  } catch (error) {
+    res.locals.error = error;
+
+    next('route');
+  }
```

Comment:
we should probably make some shared helper for this repeat patter @ some point (maybe a ssr-aas-utils package?)

I can put on the kanban

Reply:
I agree. This code also lowered the test coverage...

Reply:
ha -- we can ignore coverage report for now -- If you want, maybe write tests for everything in this PR? (if possible)

Reply:
Yeah, I'll add some tests.